src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Browser.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        public static bool IsOSPlatform(OSPlatform osPlatform) => osPlatform.Equals(OSPlatform.Browser);
+
+        public static string OSDescription => "web";
```

Comment:
@stephentoub any suggestion for a better name?

Reply:
if we're using this for webassembly, "web" is a reasonable choice but it may cause confusion since webassembly apps are being hosted already in non-web environments

if we have a "wasm" architecture then the os enumeration for web browsers should probably be "browser" because "web" is more general

Reply:
This is for `browser-wasm` only

Reply:
If the OS here is considered "Browser", can we get / return any details that describe the browser being used, just as happens on Windows, Linux, macOS, etc.?

Reply:
Browser implementors/standards people are moving towards a model where browser fingerprinting capabilities are kept to a minimum and only available on trusted origins, so you shouldn't count on being able to probe any details about the browser. The data available right now is going to be falsified in the future

Reply:
At a minimum then I think I'd prefer "Browser" instead of "web", given the OSPlatform is named Browser.